### PR TITLE
fix: add instance type

### PR DESCRIPTION
**Pull request by azayz**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

File: now/app/search_app/app.py [hunk 1]
```diff
@@ -119,7 +120,8 @@ def sbert_encoder_stub() -> Tuple[Dict, int]:
                 'model_name': 'msmarco-distilbert-base-v3',
             },
             'jcloud': {
-                'autoscale': {'min': 0, 'max': 5, 'metric': 'concurrency', 'target': 1}
+                'autoscale': {'min': 0, 'max': 5, 'metric': 'concurrency', 'target': 1},
+                'instance': 'C6',
```
Comment (deepankarm) [inline now/app/search_app/app.py:124]:
```suggestion
                'resources': {'instance': 'C6'},
```

File: now/app/search_app/app.py [hunk 2]
```diff
@@ -89,7 +89,8 @@ def preprocessor_stub(testing=False) -> Dict:
                     'max': 100,
                     'metric': 'concurrency',
                     'target': 1,
-                }
+                },
+                'instance': 'C2',
```
Comment (deepankarm) [inline now/app/search_app/app.py:93]:
```suggestion
                resources': {'instance': 'C2'},
```

File: now/app/search_app/app.py [hunk 3]
```diff
@@ -170,11 +172,7 @@ def indexer_stub(
                     'app': 'indexer',
                     'provision-index': provision_index,
                 },
-                'resources': {
-                    'memory': '8G',
-                    'cpu': 0.5,
-                    'capacity': 'on-demand',
-                },
+                'instance': 'C6',
```
Comment (deepankarm) [inline now/app/search_app/app.py:175]:
```suggestion
                resources': {'instance': 'C6'},
```

File: now/common/flow/flow-external-clip.yml [hunk 4]
```diff
@@ -19,6 +19,4 @@ executors:
       JINA_LOG_LEVEL: DEBUG
     jcloud:
       expose: true
-      resources:
-        memory: 5G
-        gpu: '1'
\ No newline at end of file
+      instance: G1
```
Comment (deepankarm) [inline now/common/flow/flow-external-clip.yml:22]:
```suggestion
      resources': {'instance': 'G1'},
```

**Review by deepankarm — CHANGES_REQUESTED** [review]
It should be still `jcloud.resources.instance` not `jcloud.instance`

**Review by joschkabraun — APPROVED** [review]
LGTM!! Let's merge ASAP

**Review by Tanguyabel — APPROVED** [review]

**Pull request merged**